analytical/templatetags/piwik.py

```diff
@@ -21,20 +21,22 @@
 SITEID_RE = re.compile(r'^\d+$')
 
 TRACKING_CODE = """
+<!-- Piwik -->
 <script type="text/javascript">
   var _paq = _paq || [];
   %(variables)s
   _paq.push(['trackPageView']);
   _paq.push(['enableLinkTracking']);
   (function() {
-    var u=(("https:" == document.location.protocol) ? "https" : "http") + "://%(url)s/";
+    var u="//%(url)s/";
     _paq.push(['setTrackerUrl', u+'piwik.php']);
     _paq.push(['setSiteId', %(siteid)s]);
-    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0]; g.type='text/javascript';
-    g.defer=true; g.async=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
   })();
 </script>
-<noscript><p><img src="http://%(url)s/piwik.php?idsite=%(siteid)s" style="border:0;" alt="" /></p></noscript>
+<!-- End Piwik Code -->
```

Comment:
Same here: Let's not include the (optional) HTML comments from the Piwik snippet.